qdrant-landing/content/documentation/integrations/dlt.md

```diff
@@ -0,0 +1,101 @@
+---
+title: DLT
+weight: 1300
+---
+
+# DLT(Data Load Tool)
+
+[DLT](https://dlthub.com/) is an open-source library that you can add to your Python scripts to load data from various and often messy data sources into well-structured, live datasets.
+
+With the DLT-Qdrant integration, you can now select Qdrant as a DLT destination to load data into.
+
+**DLT Enables**
+
+- Automated maintenance - with schema inference, evolution, and alerts, and with short declarative code, maintenance becomes simple.
+- Run it where Python runs - on Airflow, serverless functions, notebooks. No external APIs, backends or containers, scales on micro and large infrastructure alike.
+- User-friendly, declarative interface that removes knowledge obstacles for beginners while empowering senior professionals.
```

Comment:
Is that part extracted from their website? Sounds like some strong marketing :sweat_smile:

Reply:
Yes. From the site and docs. I toned it down along with the other changes.
https://github.com/qdrant/landing_page/pull/382/commits/25b4bd9b210912d63c4599edfa5dfff1616a6274